account_invoice_change_currency/data/ir_server_actions.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="toggle_force_custom_rate_action" model="ir.actions.server">
+        <field name="name">Toggle custom force rate</field>
+        <field name="model_id" ref="account.model_account_move" />
+        <field name="state">code</field>
+        <field name="code">
+for inv in records:
+    inv._toggle_forced_rate()</field>
+    </record>
+    <function name="create_action" model="ir.actions.server">
```

Comment:
remove, you only need to provide `binding_model_id` field inorder for the action to be bind